packages/account/src/Sections/Profile/PersonalDetails/personal-details-form.tsx

```diff
@@ -275,41 +272,42 @@ export const PersonalDetailsForm = observer(({ history }: Partial<RouteComponent
                                                     data-testid='dt_last_name'
                                                 />
                                             </InputGroup>
-                                        </DesktopWrapper>
-                                        <MobileWrapper>
-                                            <fieldset className='account-form__fieldset'>
-                                                <Input
-                                                    data-lpignore='true'
-                                                    type='text'
-                                                    name='first_name'
-                                                    id='first_name_mobile'
-                                                    label={localize('First name*')}
-                                                    value={values.first_name}
-                                                    onChange={handleChange}
-                                                    onBlur={handleBlur}
-                                                    required
-                                                    disabled={isFieldDisabled('first_name')}
-                                                    error={errors.first_name}
-                                                    data-testid='dt_first_name'
-                                                />
-                                            </fieldset>
-                                            <fieldset className='account-form__fieldset'>
-                                                <Input
-                                                    data-lpignore='true'
-                                                    type='text'
-                                                    name='last_name'
-                                                    id='last_name_mobile'
-                                                    label={localize('Last name*')}
-                                                    value={values.last_name}
-                                                    onChange={handleChange}
-                                                    onBlur={handleBlur}
-                                                    required
-                                                    disabled={isFieldDisabled('last_name')}
-                                                    error={errors.last_name}
-                                                    data-testid='dt_last_name'
-                                                />
-                                            </fieldset>
-                                        </MobileWrapper>
+                                        ) : (
+                                            <div>
```

Comment:
In  `(lines 231 & 488)`, we're already using `<Fragment>.` To  wrap 2 `<fieldset>` elements `(lines 277-292 and 293-308)` within a common container, we used  `<div>`. Nesting fragments within fragments could cause more complexity according to me?

lemme know if im wrong 😄

Reply:
Hi @Dhruv-deriv 👋 

No man fragments never cause more complexity to code. 
Its better to use `fragments` instead of `div` 

<img width="762" alt="Screenshot 2024-06-12 at 3 08 53 PM" src="https://github.com/binary-com/deriv-app/assets/121229483/16240d54-51e7-4ea0-8fc4-853872c94b0d">